src/overlays/actors/ovl_Bg_Inibs_Movebg/z_bg_inibs_movebg.h

```diff
@@ -5,9 +5,13 @@
 
 struct BgInibsMovebg;
 
+#define BGINIBSMOVEBG_GET_F(thisx) (u16)(thisx.actor.params) & 0xF
+
 typedef struct BgInibsMovebg {
-    /* 0x000 */ Actor actor;
-    /* 0x144 */ char unk_144[0x24];
+    /* 0x000 */ DynaPolyActor dyna;
+    /* 0x15C */ s32 unk_15C;
+    /* 0x160 */ s32 unk_160;
+    /* 0x164 */ s32 unk_164;
```

Comment:
```suggestion
    /* 0x15C */ Gfx* unk_15C;
    /* 0x160 */ Gfx* unk_160;
    /* 0x164 */ AnimatedMaterial* unk_164;
```
Based on the function calls in `BgInibsMovebg_Draw` we can deduce these types.